### PR TITLE
Metals boss reso correction

### PR DESCRIPTION
**Pull request by Cyeche**
Compute metal 1D power spectrum
Add correction of spectrograph resolution

### CODE REVIEW

File: bin/do_Pk1D.py [hunk 1]
```diff
@@ -131,10 +133,21 @@ def compute_mean_delta(ll,delta,iv,zqso):
...
+        # control histograms
+        if (args.forest_type=='Lya'):
+            forest_inf=1040.
+            forest_sup=1200.
+        elif (args.forest_type=='SiIV'):
+            forest_inf=1270.
+            forest_sup=1380.
+        elif (args.forest_type=='CIV'):
+            forest_inf=1410.
+            forest_sup=1520.
```
Comment (londumas) [inline bin/do_Pk1D.py]: why not read forest_inf and forest_sup from parser.parse_args()?

File: py/picca/data.py [hunk 2]
```diff
@@ -161,7 +161,7 @@ def __init__(self,ll,fl,iv,thid,ra,dec,zqso,plate,mjd,fid,order,diff=None,reso=N
 
         # compute means
         if reso is not None : self.mean_reso = sum(reso)/float(len(reso))
-
+
```
Comment (londumas) [inline py/picca/data.py]: trailing spaces

File: bin/do_Pk1D.py (hunk 1 above)
Comment (Cyeche) [reply bin/do_Pk1D.py]: It is just for the histograms, by definition it is the maximal width of the forest, I want that the limits are fixed. By this way I can see if in do_deltas I have used a narrower forest

File: py/picca/data.py (hunk 2 above)
Comment (Cyeche) [reply py/picca/data.py]: Ouais I know

**Pull request merged**